src/GNSSSignals.jl

```diff
@@ -29,6 +31,7 @@ module GNSSSignals
         get_carrier_amplitude_power,
         get_subcarrier_frequency,
         get_code_spectrum,
+        get_system_string,
```

Comment:
Is this exported intentionally? I think this should not be exported necessarily.
BTW: I see `fpcarrier!` and `fpcarrier_phases!` are exported even though they are not implemented anymore.

Reply:
Yes, this was exported intentionally, as there are some situations, where the name of the used GNSS must be accessed without the full type specification. We could also implement the `string` method, which currently gives the same output as `show`.